### ISSUE TITLE
Add support for showing full screen logs when push hooks are running

### ISSUE THREAD

**Issue by ytkimirti** [jesseduffield/lazygit#3717]
I usually have hooks on commit and push actions. When committing, lazygit ui hides, and shows only the output of the hook (linting process etc.) only, and if it gives an error you can see it clearly.

But when I push, I can only see the output from the small command log screen. It's not colored and I can't do search using the vscode terminal. For example, I have a hook that builds the nextjs project before pushing and it's really hard to work with the large error output from that small tui window.

If it had the same behaviour as commit, it would be really usefull.

**Comment by zihaolam** [comment]
have you tried pressing '@' and selecting 'Focus command log' during commits/push, it displays the command outputs in a more maximized section

**Comment by ytkimirti** [comment]
Yes, but I would prefer the full terminal. For example when linting the project on push, if there is a linting error it shows it in a popup. The coloring works well but
- it is smaller
- the scroll works but not like the native scroll
- clicking file links to open the problematic file doesn't work
- searching on the terminal buffer does not work since only thing the terminal can see is the UI

**Comment by mcky** [comment]
It does feel like if the error message is telling me to use `@`, it could give me a prompt to focus the command log straight away

I'd also love to be able to map issues in hooks to files in the file panel, kind of like how github actions can be configured with annotations - but that's a separate issue, and obviously anything tool-specific is way out of scope of lazygit itself.

Also +1 that the tool generates clickable links when I run it standalone, but it loses that functionality when in the command log panle
